CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 ## master / unreleased
+ - [ENHANCEMENT] Time-ovelapping blocks are now allowed.
```

Comment:
After another look at CHANGELOG, I think we should start enforcing adding the PR number too with the entry, so that it is easy to jump to the PR when needed (Like I did for empty blocks here https://github.com/prometheus/tsdb/blob/master/CHANGELOG.md#040).

Reply:
yep that should be ok.